scripts/34-Plain-Text-Team.md

```diff
@@ -0,0 +1,601 @@
+
+## &mdash; Adam Savage
+
+notes:
+
+Human memory is extremely unreliable, subjective, and the root cause of many problems.
+
+After a discussion, it is not apparent that everyone has agreed upon exactly the same thing.
+And you now need another meeting to double-check that.
+
+The solution is documentation.
+
+---
+
+# Documentation-First Teams
+
+notes:
+Everyone understands this, from 10,000-page government specifications to an email sign-off from the client you're making a 3-minute track for.
+Communication is most reliable when it is in black and white.
+
+Yes, have more immediate conversations, by video, or chat, but write down what you concluded, and get the other person to confirm it.
+
+---
+
+# ["Documenting Architecture Decisions"](https://cognitect.com/blog/2011/11/15/documenting-architecture-decisions)
+
+(aka the _ADR_ process)
+
+&mdash; [Michael Nygard](https://cognitect.com/authors/MichaelNygard.html)
+
+notes:
+You can improve every part of your team, business, or organisation by recording what decisions you have made, and WHY, in a system that allows for asynchronous discussion and improvements.
+
+The ADR process is excellent for this, for example.
+
+There are a thousand competing apps that claim to solve these problems for you.
+
+---
+
+<split even>
+
+![[gdocs-screenshot.png|200]]
+
+![[jamboard-photo.png|200]]
+
+![[confluence-screenshot.png|200]]
+
+![[pivotal-tracker-screenshot.png|200]]
+
+![[notion-screenshot.png|200]]
+
+![[trello-screenshot.png|200]]
+
+</split>
+
+notes:
+These apps all re-invent the wheel in their own way, and new ones are being released every week.
+I've used most of them, perhaps you have too, and they're all rubbish.
+
+But there is a group of people who are extremely practiced at managing enormous distributed, concurrent, text projects:
+
+*Programmers!*
+
+As an example, if you use Google Docs, your small team can collaborate on a few files a day, in a drive of perhaps a hundred or two hundred.
+And just like in most other documentation systems, that won't scale.
+
+Programmers simultaneously edit thousands of files a day, across repositories of data so numerous that we don't keep count.
+
+What are programmers using, and can non-programmers use it too?
+
+---
+
+# Enter Git
+
+- GitHub
+- GitLab
+- Bitbucket
+- SourceForge
+- Etc.
+
+notes:
+
+The answer is yes, yes we can.
+
+I recommend you use the most popular distributed version control system on the planet: Git.
+
+You'll use this through one of the many git web hosts, the largest of which is GitHub, which I recommend for most people.
+
+---
+
+# Popularity Matters
+
+notes:
+
+Though I mention GitHub primarily in this video, I'm not sponsored by them, I just acknowledge that popularity matters.
+Support, experience, and integrations with other services will all be far, far easier if you use the standard.
+
+All these tools started as a web interface around the incredible tool: Git.
```

Comment:
I think the wording you used above, "one of the many git web hosts, the largest of which is GitHub", is perfectly balanced in supporting pluralism while recommending the obvious tool.

Given that, I don't feel this slide is necessary. It appears to me to be putting more of a thumb on the scale for GitHub, even though I think your intention is the opposite. I would simply remove this slide if I were you.

Reply:
It's clunky because I need to be hyper clear about my lack of sponsorship from GitHub, and yet my recommendation is still GH

Reply:
You do mention it in a slide lower down... It feels a bit overly anxious to me. Where are you getting this need to hyper clarity?